bangazon/bangazon_ui/tests/test_order_model.py

```diff
@@ -0,0 +1,57 @@
+import sys
+sys.path.append("../")
+from django.test import TestCase
+from django.contrib.auth.models import User
+from bangazon_ui.models.customer_model import Customer
+from bangazon_ui.models.payment_type_model import PaymentType
+from bangazon_ui.models.product_model import Product
+from bangazon_ui.models.order_model import Order
+
+
+class TestOrderModel(TestCase): 
+
+    """
+    This is a test to build implementation code for the Order Model. 
+
+    """
+
+def test_order(self):
+
+    user = User.objects.create_user(
+    first_name="Dani",
+    last_name="Dani",
+    username="danidani",
+    password="asdf1234")
+
+    customer = Customer.objects.create(
+    user=user,
+    address="3040 NSS Drive",
+    city="Nashville",
+    state="TN",
+    zip_code="12345",
+    phone="123456789")
+
+    payment = PaymentType.objects.create(
+    customer=customer,
+    payment_type="Visa",
+    account_number="12345678",
+    ccv="111",
+    expiration_date="2017-01-01"
+    )
+
+    product = Product.objects.create(
```

Comment:
Here are the relevant fields, with the suggested changes that let it work for me.
```
name = models.CharField(max_length=100, blank=True, default="")
price = models.DecimalField(max_digits=20, decimal_places=2)
description = models.TextField(max_length=300, default='')
quantity = models.IntegerField()
product_type =models.ForeignKey(ProductType, related_name="products", on_delete=models.CASCADE)
customer =models.ForeignKey(Customer, related_name="seller", on_delete=models.CASCADE)
```